apps/studio/components/interfaces/Storage/Storage.constants.ts

```diff
@@ -65,20 +65,25 @@ export const CONTEXT_MENU_KEYS = {
 export const BUCKET_TYPES = {
   files: {
     displayName: 'Files',
+    bucketName: 'a file bucket',
     description: 'General file storage for most types of digital content.',
+    valueProp: 'Store images, videos, documents, and any other file type.',
     docsUrl: `${DOCS_URL}/guides/storage/buckets/fundamentals`,
   },
   analytics: {
     displayName: 'Analytics',
+    bucketName: 'an analytics bucket',
     description: 'Purpose-built storage for analytical workloads.',
+    valueProp: 'Store large datasets for analytics and reporting.',
     docsUrl: `${DOCS_URL}/guides/storage/analytics/introduction`,
   },
   vectors: {
     displayName: 'Vectors',
+    bucketName: 'a vector bucket',
```

Comment:
English is weird with articles(“an” vs “a“) based on subsequent word sound. Ditto for whether a word should be plural.

Clearer (I think) to write this explicitly as a constant for each than to have weird button copywriting like “Create files bucket”